src/lib/free-queue/README.md

```diff
@@ -1 +1,114 @@
-### Work In Progress ....
\ No newline at end of file
+# FreeQueue: Ring-Buffer Library for AudioWorklet
+
+## Introduction
+
+This library provides two classes, `FreeQueue` and `FreeQueueSAB`, designed to handle audio buffering using WebAssembly (WASM) and SharedArrayBuffer (SAB) respectively. These classes offer efficient ways to manage audio data, making them suitable for applications requiring low-latency audio processing.
+
+A ring buffer is particularly beneficial in audio applications because it provides flexibility and smooth integration with other system components. Although a ring buffer may introduce more latency compared to non-buffering solutions, a well-implemented ring buffer minimizes this latency while maintaining efficient data handling.
+
+### Features
+
+- Efficient audio buffering using WASM (`FreeQueue`) and SharedArrayBuffer (`FreeQueueSAB`).
```

Comment:
Small nit: I think it's more clear to say "using WASM (`FreeQueue`) **or** SharedArrayBuffer"